### DISCUSSION TITLE
Issue occured while fetching data from tables

### DISCUSSION THREAD

**Discussion by vinayBhoure** [supabase/supabase#19652] [Questions]
I'm not able to fetch data from tables using javascript. It is returning an error( type error: failed to fetch). I'm new to Supebase, how could I fix this error?

**Comment by GaryAustin1** [comment]
You will need to provide more info on your code.  That is a very broad error and usually means the request is not even going out to Supabase. 
You might also consider the Supabase Discord for help as it is more interactive for this type of help if it requires a bunch of back and forth.

**Comment by vinayBhoure** [comment]
Thank Gary for replying. Issue was resolved, I stucked because of some row level security which I fixed.
